src/components/AsyncButton/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { AsyncButtonStyles } from '../../styles/AsyncButton';
+
+const AsyncButton = ({ type, onClick, loading }) => {
```

Comment:
my be this better
```
const AsyncButton = ({ type, onClick, loading }) => (
  <AsyncButtonStyles
        disabled={loading}
        type={type}
        onClick={onClick}
      >
        {loading ? 'Submitting...' : 'Submit'}
      </AsyncButtonStyles>
)
```

Reply:
@gianglevan94 Yeah you're right. Thanks 💯